recipes/pylint-common/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pylint-common" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "0abe78dc9259927fcad69f2f8220f8c70da62d32bf8393b8768f5ef6c9870e11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  {% set pypi_name = name.replace("_", "-") %}
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pylint >=1.0
+    # Was not explicitly listed.
+    # However it is being imported.
+    # So have added it as a dependency.
+    # See linked issue for details.
+    #
+    # https://github.com/landscapeio/pylint-common/issues/2
+    #
+    - pylint-plugin-utils
+
+  run:
+    - python
+    - pylint >=1.0
+    # Was not explicitly listed.
+    # However it is being imported.
+    # So have added it as a dependency.
+    # See linked issue for details.
+    #
+    # https://github.com/landscapeio/pylint-common/issues/2
+    #
+    - pylint-plugin-utils
+
+test:
+  imports:
+    - pylint_common
+
+about:
+  home: https://github.com/landscapeio/pylint-common
+  license: GPL 2
```

Comment:
Proposing the license file be added to the manifest in PR ( https://github.com/landscapeio/pylint-common/pull/4 ). May need to resolve this in the feedstock.